src/sass_context.cpp

```diff
@@ -189,12 +189,18 @@ extern "C" {
 
   static void copy_strings(const std::vector<std::string>& strings, char*** array) {
     int num = static_cast<int>(strings.size());
-    char** arr = (char**) malloc(sizeof(char*) * (num + 1));
-    if (arr == 0) throw(std::bad_alloc());
+    char** arr = (char**) calloc(num + 1, sizeof(char*));
+    if (arr == 0)
+      return;
```

Comment:
Sure, my mistake. Even more, let's make it return the `(char **)` pointer for easier error checking.